Documentation/ApiOverview/ContentElements/_AddingYourOwnContentElements/_tt_content.php

```diff
@@ -1,29 +1,32 @@
 <?php
 
 declare(strict_types=1);
+use TYPO3\CMS\Core\Utility\ExtensionManagementUtility;
+
 defined('TYPO3') or die();
 
-// Adds the content element to the "Type" dropdown
-\TYPO3\CMS\Core\Utility\ExtensionManagementUtility::addTcaSelectItem(
-    'tt_content',
-    'CType',
-    [
-        // title
-        'label' => 'LLL:EXT:my_extension/Resources/Private/Language/locallang.xlf:myextension_newcontentelement_title',
-        // plugin signature: extkey_identifier
-        'value' => 'myextension_newcontentelement',
-        // icon identifier
-        'icon' => 'content-text',
-        // group
-        'group' => 'default',
-        // description
-        'description' => 'LLL:EXT:my_extension/Resources/Private/Language/locallang.xlf:myextension_newcontentelement_description',
-    ],
-    'textmedia',
-    'after',
-);
+call_user_func(function () {
+
+    $key = 'examples_basiccontent';
```

Comment:
This key does not match value `examples_newcontentelement` in documentation.

Quote: "We use examples_newcontentelement throughout the simple example."

Reply:
I unified the key to myextension_basiccontent all over, as we also call the extension my_extension by default